lib/sidekiq-scheduler/redis_manager.rb

```diff
@@ -106,7 +106,7 @@ def self.schedule_exist?
     #
     # @return [Array] array with all the changed job names
     def self.get_schedule_changes(from, to)
-      Sidekiq.redis { |r| r.zrangebyscore(schedules_changed_key, from, "(#{to}") }
+      SidekiqScheduler::SidekiqAdapter.redis_zrangebyscore(schedules_changed_key, from, "(#{to}")
```

Comment:
I... assume the existing argument is ok? 🤷‍♂️

Reply:
yeah don't know why we have the `(` lol, but let's keep it